src/main.rs

```diff
@@ -22,6 +22,9 @@ mod templates;
 use crate::config::{Config, DEFAULT_ENV_NAME, DEFAULT_PROJ_NAME};
 use crate::environments::Environments;
 use crate::graphql::GraphQLError;
+use crate::parameters::export_parameters_query::{
+    ExportParametersFormatEnum, ExportParametersOptions,
+};
```

Comment:
Nice! This was the missing piece for me and why I could never get the options passed into the method call.